packages/transport-native/README.md

```diff
@@ -0,0 +1,20 @@
+# trezor-transport-native
+
+Trezor Transport layer for low level communication with Trezor devices for React Native.
+
+## Installation
+
+```sh
+yarn add trezor-transport-native
```

Comment:
`trezor-transport-native` stayed in several places instead of `@trezor/transport-native`

Reply:
reamed everywhere